### PR TITLE
NL : Add Private TV Italia

### PR DESCRIPTION
**Pull request by BellezaEmporium**
Had a Polish version, but it's protected by Nagravision encryption.

Also had Brazzers HD, but it's protected by Nagravision too.

image

### CODE REVIEW

File: channels/us.m3u [hunk 1]
```diff
@@ -3695,3 +3695,5 @@ https://rockentertainment-zoomoo-1.samsung.wurl.com/manifest/playlist.m3u8
 http://188.40.68.167/russia/disney/playlist.m3u8
 #EXTINF:-1 tvg-id="MeiGuoQieDaDianShi.us" tvg-name="美国切达电视" tvg-country="US" tvg-language="" tvg-logo="" group-title="",美国切达电视 (1080p)
 https://live.chdrstatic.com/cheddar/primary/4.m3u8
+#EXTINF:-1 tvg-id="PrivateTV.us" tvg-name="Private TV (Italia)" tvg-country="US" tvg-language="" tvg-logo="" group-title="XXX",Private TV (Italia)
```
Comment (freearhey) [inline channels/us.m3u:3698]: This is a channel from the Nederlands: https://www.lyngsat.com/tvchannels/nl/Private-TV.html

**Review by freearhey — CHANGES_REQUESTED** [review]

File: channels/us.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/us.m3u:3698]: Noted, Moving it when I can

**Comment by LaneSh4d0w** [comment]
Should be good.

File: channels/nl.m3u [hunk 2]
```diff
@@ -203,3 +203,5 @@ https://xite-samsung-de.amagi.tv/playlist.m3u8
 https://ms7.mx-cd.net/tv/226-1855782/XITE_NL.smil/chunklist_w323364313_b3328000_slNLD.m3u8
 #EXTINF:-1 tvg-id="ZO34.nl" tvg-name="ZO!34" tvg-country="NL" tvg-language="" tvg-logo="https://i.imgur.com/KrEBHLw.jpg" group-title="",ZO!34 (720p)
 https://streams.uitzending.tv/zo34/zo34/playlist.m3u8
+#EXTINF:-1 tvg-id="PrivateTV.nl" tvg-name="Private TV (Italia)" tvg-country="US" tvg-language="" tvg-logo="" group-title="XXX",Private TV (Italia)
```
Comment (freearhey) [inline channels/nl.m3u:206]: The value of `tvg-country` does not seem to be correct.

**Review by freearhey — CHANGES_REQUESTED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**